lib/read.js

```diff
@@ -335,6 +356,13 @@ class ReadSql extends AdapterRead {
                     readEnd: this.timeField ? to : new JuttleMoment(Infinity)
                 };
             }
+            
+            if (this.sortFields) {
```

Comment:
I guess I don't understand what this is doing...if we read with a limit of `X` points a time range with `2X` points in it, and optimize a `sort` on that, won't we return only the first `X` points but set the `readEnd` to `t2` and never get the second page?